.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: go
 sudo: required
 go:
-- 1.8
-- 1.9
+- 1.11
```

Comment:
Can this be ` - "1.11.x"` so that we pick up the latest point release instead of just `1.11`?

Reply:
Lets try.